import main

try:
    main.main()
except:
    print("error")
